src/compare_covermaps.py

```diff
@@ -512,55 +551,63 @@ def generate_report(dataset_name: str, country: str, true, pred) -> pd.DataFrame
         )""",
         resolution=10,
         crop_labels=[100],
+        years_covered=[2020],
     ),
     "glad": Covermap(
         "glad",
         'ee.ImageCollection("users/potapovpeter/Global_cropland_2019")',
         resolution=30,
         probability=0.5,
+        years_covered=[2003, 2007, 2011, 2015, 2019],
```

Comment:
This collection only includes images for 2019

Reply:
Oh yeah, this one is weird. I'll make it so that it changes the collection.